syntax = "proto3";
option csharp_namespace = "TomTaw.eWordIMCIS.Proto";
import "UserOrgInfoProto.Proto";

message DocumentInputProto{
string examUID=1;
string typeCode=2;
string classCode=3;
string businessType=4;
string access_Token=5;
string shareAddress=6;
string shareUserName=7;
string sharePassWord=8;
int32 pageSize=9;
int32 currentPage=10;
string token=11;
string isInteragency =12;
string isInteragencyExam =13;
string isInteragencyClinic =14;
string isInteragencyStatistic =15;
UserOrgInfoProto userInfo =16;
string isInteragencyData =17;
string applyMethod =18;
string isInterconnectData=19; 
string accessionNumber=20;
string organizationID=21;
string examToken=22; //病人token，分享病例带的token
}